src/pages/ProfilePage.tsx

```diff
@@ -265,6 +268,9 @@ export default withOnyx<ProfilePageProps, ProfilePageOnyxProps>({
     personalDetails: {
         key: ONYXKEYS.PERSONAL_DETAILS_LIST,
     },
+    personalDetailsMetadata: {
+        key: ONYXKEYS.PERSONAL_DETAILS_METADATA,
```

Comment:
Do we need to connect to the whole thing, or can we use a selector to just connect to the one accountID we're interested in?